### ISSUE TITLE
feature request: improve documentation on using foulders (safely) outside /home/yunohost.app/syncthing/

### ISSUE THREAD

**Issue by ahcheing** [YunoHost-Apps/syncthing_ynh#156]

**Comment by Pfeil** [comment]
I agree this should be documented. In order to avoid breaking the existing setup somehow, I used the ACL instead of groups and users/owners. I think I did something like this (it has been a while):

`sudo setfacl -Rm user:syncthing:rw /home/yunohost.multimedia/share`

With this, the basic groups and owners stay as it is. Not sure if this is the best thing to do, but from my understanding, this is the minimal change we need to do to make it work. If people agree to this solution, if you add such an example to the app information shown in the yunohost app management interface, I think this issue would be solved.

**Comment by agdamsbo** [comment]
> ```sudo setfacl -Rm user:syncthing:rw /home/yunohost.multimedia/share```

So I have this same issue. I want to sync a folder in /home/yunohost.multimedia/share to a folder on a different machine. I keep getting errors trying to setup Syncthing to sync anything outside /home/yunohost.app/syncthing. I have tried your suggestion to edit the ACL only, no luck. I have also tried to add the syncthing user to the root group. No luck either. What to do? Thank you very much.

**Comment by Pfeil** [comment]
Are you sure you restarted syncthing after changing the ACL? If so, maybe there is any hint in the logs? It is still working for me. What does `getfacl /home/yunohost.multimedia/share` give you about the syncthing user?
